magento/ece-tools
# MCLOUD-6656: Synchronize list of available ES versions in all sources where it is mentioned

### Description

There is discrepancy between the lists of available ES versions mentioned in ece-tools (or devdocs)  and versions which Platform supports on different environments.
(ES versions 6.8 and 7.6 are not supported on Pro environments)

### Fixed Issues (if relevant)

1. https://jira.corp.magento.com/browse/MCLOUD-6656

### Manual testing scenarios

- Set ES version as 6.8/7.6
- After the deployment see warning in logs that this version is not supported

### Release notes

For user-facing changes, add a meaningful release note. For examples, see [Magento Cloud ECE-tools release notes](https://devdocs.magento.com/cloud/release-notes/ece-release-notes.html).

### Associated documentation updates

Add link to Magento DevDocs PR or Issue, if needed.

### Contribution checklist
